l10n_it_ddt/views/stock_picking_package_preparation.xml

```diff
@@ -151,10 +155,40 @@
                             <field name="carrier_id"/>
                             <field name="parcels"
                                    attrs="{'required':[('ddt_type_id', '!=', False)]}"/>
-                            <field name="volume"/>
+                            <label for="volume"/>
```

Comment:
Per questo e i successivi, puoi risparmiare un bel po' di righe facendo come in https://github.com/odoo/odoo/blob/ee65b44d9e51f380e1b6978485e466b5720df215/addons/stock/views/stock_scrap_views.xml#L49-L52

Reply:
Grazie, ho anche migliorato un po' la distribuzione dei campi